templates/CRM/common/l10n.js.tpl

```diff
@@ -115,6 +115,7 @@
     if ($('#crm-notification-container').length) {
       $.each(validator.errorList, function(k, error) {
         $(error.element).parents('.crm-custom-accordion.collapsed').crmAccordionToggle();
```

Comment:
Should this line be removed?  Looks like it is being replaced by the new line below.

Reply:
My thinking was backwards compatibility. I wasn't sure how widely this was used. Does this affect extensions too?

Reply:
Ok, makes sense.

Reply:
It could be good to maybe add a comment in there indicating that we should remove it but whatever.